styles/modal.css

```
@@ -17,72 +28,185 @@ body.modal-open {
-  opacity: 1;
-}
-
-.modal {
-  background-color: var(--color-bluegray-90);
-  padding: 20px;
-  border-radius: 16px;
-  color: white;
-  z-index: 2;
-  position: relative;
-  width: 1000px;
+  /*transition: opacity 0.3s ease, visibility 0.3s ease;*/
```

Stylelint (17.6.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)